src/test/app/common/calculate-month-increment/calculate-month-increment.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from 'chai'
+
+import { calculateMonthIncrement } from 'common/calculate-month-increment/calculate-month-increment'
+import * as moment from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+
+describe('calculateMonthIncrement', () => {
+
+  it('should calculate a month where the start date is before the 28th',() => {
```

Comment:
Good coverage of different scenario's for dates. may want to look at following years (2019) I see you have done for 2020.